src/user/scheduling.md

```diff
@@ -319,3 +319,99 @@ Some aspects of the meta data synchronization feature to be aware of:
 
 - You can see the time of last successful synchronization with remote
   server in the scheduling screen next to the "Last success" label.
+
+### Predictor { #scheduling_predictor } 
+
+This runs selected predictors and/or predictor groups.
+
+The relative start and relative end parameters determine the periods in
+which data will be predicted, relative to the date on which the
```

Comment:
The three "relatives" are accurate, but they make it harder for me to read.

Maybe skip the second one and change the third to "corresponding", i.e.:
 > The relative start and end parameters determine the periods in
which data will be predicted, corresponding to the date on which the predictor job is run:

Reply:
Yeah, I have a lot of relatives. Fixed.

Reply:
Looks good, thanks!